### ISSUE TITLE
Update Gemini staging fee address and AC address `recipient_id`

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#21648]
## Description

- [x] Update Gemini staging fee address
- [x] Update Gemini staging AC address 

cc: @LaurenWags

**Comment by kjozwiak** [comment]
Adding missing `1.38.x` milestone as this landed in `master` which is currently `1.38.x`.

**Comment by kjozwiak** [comment]
Will require `1.37.96` or higher for verifications on `1.37.x`.

**Comment by LaurenWags** [comment]
Labelling as `QA/Blocked` as there's no way to confirm this until we get https://github.com/brave/brave-browser/issues/21591 into 1.37.x.

**Comment by LaurenWags** [comment]
Removing `OS/Android` as Gemini is not available on Android with 1.37.x and this will be covered as part of testing when Gemini is tested on Android.

**Comment by LaurenWags** [comment]
Removed `QA/Blocked` as https://github.com/brave/brave-browser/issues/21591 is available on 1.37.103 and above.

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```
Brave | 1.37.103 Chromium: 100.0.4896.46 (Official Build) beta (64-bit)
-- | --
Revision | 5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}
OS | Windows 10 Version 21H2 (Build 19044.1586)

```

- Verified that 95% of tip and 5% of tip (fee) were contributed as expected.
- Verification notes are linked in the issue https://github.com/brave/brave-browser/issues/21591#issuecomment-1076149183
